storage/namespace.go

```diff
@@ -495,6 +498,17 @@ func (n *dbNamespace) WriteTagged(
 	return err
 }
 
+func (n *dbNamespace) QueryIDs(
+	ctx context.Context,
+	query index.Query,
+	opts index.QueryOptions,
+) (index.QueryResults, error) {
+	callStart := n.nowFn()
+	res, err := n.reverseIndex.Query(ctx, query, opts)
```

Comment:
Hm this will panic at this point as `reverseIndex` is nil, should we instead for now `return fmt.Errorf("not implemented")` or something?

Reply:
This can't happen right now because the only caller to this method is `db.QueryIDs` which does the equivalent check before calling `ns.QueryIDs`.

Reply:
Added an error check and log statement to be defensive.